### PR TITLE
Computer vision api 2.1

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

**Comment by jianghaolu** [comment]
This is generated by the Track 1 /Milis generator to keep it backward compatible with 2.0 API version. Computer vision will move to Track 2 generator for their 3.0 version.

File: pom.data.xml [hunk 1]
```diff
@@ -605,7 +605,6 @@
   <modules>
     <module>./sdk/batch/microsoft-azure-batch</module>
     <module>./sdk/applicationinsights/microsoft-azure-applicationinsights-query</module>
-    <module>./sdk/cosmos</module>
```
[the remark this answers is not in the document]
Comment (jianghaolu) [reply pom.data.xml]: Cosmos data plane is removed in the repo and the builds have been failing already.

File: sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVision.java [hunk 2]
```diff
@@ -69,344 +107,154 @@
...
+     * @param tagImageInStreamOptionalParameter the object representing the optional parameters to be set before calling this API
      * @throws IllegalArgumentException thrown if parameters fail the validation
      * @throws ComputerVisionErrorException thrown if the request is rejected by server
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent
-     * @return the ImageDescription object if successful.
+     * @return the TagResult object if successful.
      */
-    @Deprecated
-    ImageDescription describeImageInStream(byte[] image, DescribeImageInStreamOptionalParameter describeImageInStreamOptionalParameter);
+    TagResult tagImageInStream(byte[] image, TagImageInStreamOptionalParameter tagImageInStreamOptionalParameter);
```
[the remark this answers is not in the document]
Comment (jianghaolu) [reply sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVision.java]: Added back

File: sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVision.java [hunk 3]
```diff
@@ -433,108 +282,65 @@
...
-
-            /**
-             * Execute the request asynchronously.
-             *
-             * @return the observable to the OcrResult object
-             */
-            Observable<OcrResult> executeAsync();
-        }
-    }
+    Observable<DomainModelResults> analyzeImageByDomainInStreamAsync(String model, byte[] image, AnalyzeImageByDomainInStreamOptionalParameter analyzeImageByDomainInStreamOptionalParameter);
```
[the remark this answers is not in the document]
Comment (jianghaolu) [reply sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVision.java]: Added back

File: sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/models/ComputerVisionErrorCodes.java [hunk 4]
```diff
@@ -1,86 +0,0 @@
...
-
-package com.microsoft.azure.cognitiveservices.vision.computervision.models;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
-
-/**
- * Defines values for ComputerVisionErrorCodes.
- */
-public enum ComputerVisionErrorCodes {
```
[the remark this answers is not in the document]
Comment (jianghaolu) [reply sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/models/ComputerVisionErrorCodes.java]: Added "type": "string" and regenerated this.

File: sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVisionClient.java [hunk 5]
```diff
@@ -51,44 +51,44 @@
...
-     * Gets Gets or sets the preferred language for the response..
+     * Gets Gets or sets the preferred language for the response.
      *
      * @return the acceptLanguage value.
      */
     String acceptLanguage();
 
     /**
-     * Sets Gets or sets the preferred language for the response..
+     * Sets Gets or sets the preferred language for the response.
```
Comment (alzimmermsft) [inline sdk/cognitiveservices/ms-azure-cs-computervision/src/main/java/com/microsoft/azure/cognitiveservices/vision/computervision/ComputerVisionClient.java]: Do we care that these Javadocs aren't grammatically correct?

**Pull request merged**